### DISCUSSION TITLE
Error: Configuration File Not Found

### DISCUSSION THREAD

**Discussion by eTalarico** [qdm12/gluetun#1015] [Q&A] [answered]
Hello! 

I'm attempting to run an OpenVPN connection using a configuration file, but I'm getting the following error when I try to start the container: 

```
ERROR VPN settings: provider settings: server selection: OpenVPN server selection settings: configuration file: file does not exist: /gluetun/custom.conf
```

I'm following the instructions [here](https://github.com/qdm12/gluetun/wiki/OpenVPN-configuration-file#Implicit-behavior). Based on the error message, I'm pretty sure I'm doing something wrong with step 2. I simply moved the config file to be in `/gluetun/custom.conf`, and then referenced it in the variable path. Is there something else I need to do here?  

I'm then running: 

```
docker run -it --rm --cap-add=NET_ADMIN -e VPN_SERVICE_PROVIDER=custom -e OPENVPN_CUSTOM_CONFIG=/gluetun/custom.conf -v /gluetun/custom.conf:/gluetun.custom.conf:ro qmcgaw/gluetun 
```

Any help would be appreciated!

Full docker startup log: 

```
> 2022-06-06T11:42:19Z INFO [routing] default route found: interface eth0, gateway 172.17.0.1 and assigned IP 172.17.0.2
> 2022-06-06T11:42:19Z INFO [routing] local ethernet link found: eth0
> 2022-06-06T11:42:19Z INFO [routing] local ipnet found: 172.17.0.0/16
> 2022-06-06T11:42:19Z INFO [firewall] enabling...
> 2022-06-06T11:42:19Z INFO [firewall] enabled successfully
> 2022-06-06T11:42:19Z INFO [storage] creating /gluetun/servers.json with 11559 hardcoded servers
> 2022-06-06T11:42:19Z ERROR VPN settings: provider settings: server selection: OpenVPN server selection settings: configuration file: file does not exist: /gluetun/custom.conf
> 2022-06-06T11:42:19Z INFO Shutdown successful
```

**Comment by qdm12** [comment, marked as answer]
Your bind mount destination is wrong `/gluetun.custom.conf` should be `/gluetun/custom.conf`.

Ideally you should bind mount a directory `/your/path:/gluetun` where you have `/your/path/custom.conf` on your host. You also don't need to have it at /gluetun/custom.conf on your host file system.

**Reply by eTalarico** [reply]
Thanks, Quentin! That fixed it!
